setup.cfg

```diff
@@ -2,15 +2,15 @@
 python_files = test*.py
 # Note that the database is reused on tests, so run `pytest --create-db` before running the tests
 # if your models have changed
-addopts = --reuse-db --tb=native -p no:doctest -p no:warnings -s
+addopts = --reuse-db --tb=native -p no:doctest -p no:warnings
 norecursedirs = src/clims/legacy bin dist docs htmlcov script hooks node_modules .* {args}
 looponfailroots = src tests
 selenium_driver = chrome
 
 [flake8]
 ignore = F999,E501,E128,E124,E402,W503,E731,C901
 max-line-length = 100
-exclude = .venv/.git,*/migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,examples/*
+exclude = .venv/.git,*/migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,examples/*,src/clims/legacy,debug_toolbar
```

Comment:
Maybe later we'll want to auto-format all the legacy stuff to conform to the new standard? I don't mind this for now, though.

Reply:
Yeah no doubt. I was thinking too that the legacy stuff will be entirely revisited and autoformatted then.